### PR TITLE
Bg odds

### PR DESCRIPTION
**Pull request by guthb**
@katerebekah PR is up

### CODE REVIEW

**Comment by morecallan** [comment]
@guthb there's an issue with the method for calculating the odd numbers, the list increments even if no number is added, meaning that there will only be half as many numbers than the user's request. Your test is passing because you doubled the input number of the list (actual value) 🔢

**Comment by guthb** [comment]
@morecallan Ah i see the issue with my logic and test.  Let me correct it and push again

**Comment by morecallan** [comment]
👍

File: Mathmagician/Odds.cs [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+            List<int> _listOdds = new List<int>();
+            int i = 0; //limit based on input value
+            int j = 0; //new integer for odd values
+            
+            while (i < input)
+            {
+                
+                if (j % 2 == 1)
+                {
+                    _listOdds.Add(j);
```
Comment (katerebekah) [inline Mathmagician/Odds.cs:23]: Instead of i, could you use the .Count on _listOdds in your while loop?

**Comment by katerebekah** [comment]
👍looks good!

**Pull request merged**